src/components/dialog/OverlayFadingBackground.tsx

```diff
@@ -0,0 +1,46 @@
+import React, {useRef, useEffect, useCallback, useMemo} from 'react';
+import View from '../view';
+import {Animated} from 'react-native';
+
+interface Props {
+  dialogVisibility: boolean;
+  modalVisibility: boolean;
+  overlayBackgroundColor: string;
+}
+
+export default ({
+  dialogVisibility,
+  modalVisibility,
+  overlayBackgroundColor
+}: Props) => {
+  const fadeAnimation = useRef(new Animated.Value(0)).current;
+
+  useEffect(() => {
+    if (!dialogVisibility) {
+      animateFading(0);
+    }
+  }, [dialogVisibility]);
```

Comment:
The hooks plugin says you are missing `animateFading`

Reply:
fixed the warnings..
But I gotta say, the plugin is not that smart. 
It just blindly check what's being used and ask to add it as a dependency, which sometimes is not really necessary.

Reply:
I know, and it has bugs sometimes (see [AutoLockScrollView](https://github.com/wix/react-native-ui-lib/blob/master/demo/src/screens/componentScreens/WithScrollEnablerScreen/AutoLockScrollView.tsx#L41), it cannot recognize that not all the `props` need to be checked), but it did find these which are bugs, I think, so it's worth using IMHO.